src/02 - Object types.md

```diff
@@ -362,6 +364,22 @@ The following notation is used in the tables below:
+|:-----|:-------|:-------------------------|:--------:|:-----------:|:---------:|:-|:----:|:----:|:-----:|:----:|:-------------:|:--------------------------------------------------------------------------------------------|:----------|
+|      | string | id                       |    x     |      x      |           |  |  sm  |  x   |   x   |  x   |               |                                                                                             |           |
+|      | string | version                  |    x     |      x      |           |  |  sm  |  x   |   x   |      |               |                                                                                             |           |
+|      |        |                          |          |             |           |  |      |      |       |      |               |                                                                                             |           |
+| M001 | string | uuid                     |    x     |      x      |           |  |  cw  |  x   |   x   |      |               |                                                                                             |           |
+| M600 | date   | opprettetDato            |    x     |      x      |           |  |  cw  |  x   |   x   |  x   |               |                                                                                             |           |
+| M601 | string | opprettetAv              |    x     |      x      |           |  |  cw  |  x   |   x   |  x   |               |                                                                                             |           |
+| M601 | string | opprettetAvBrukerIdent   |    x     |      x      |           |  |  cw  |  x   |   x   |  x   |               |                                                                                             |           |
+| M619 | string | avskrivningsmaate        |    x     |             |     x     |  |  x   |  x   |   x   |  x   |               | <p>&bull; code list in Documaster</p>                                                       |           |
+|      |        |                          |          |             |           |  |      |      |       |      |               |                                                                                             |           |
+|      | ref    | refJournalpost           |    x     |             |           |  |      |  x   |   x   |      |       x       | may be partially supported by systems</p><p>&bull; *unlink* not supported in Documaster</p> |           |
+|      | ref    | refTilknyttetJournalpost |          |             |           |  |      |      |   x   |      |       x       | may be partially supported by systems</p><p>&bull;                                          | &nbsp;    |
```

Comment:
Unlinking `refTilknyttetJournalpost` is also not supported in Documaster.

Formatting of `Comment 1` for both refs is broken (though parsable); there are missing opening/closing paragraph tags.

Reply:
As per my previous comment - linking/unlinking `refTilknyttetJournalpost` is supported.

I have fixed the incomplete paragraph tags though.